features/step_definitions/analytics_steps.rb

```diff
@@ -39,7 +39,7 @@
 And "I search for lunch" do
   stub_search_api_request_with_query_param_no_results("lunch")
 
-  fill_in "Search", with: "lunch"
+  fill_in "Search", with: "lunch", fill_options: { clear: :backspace }
```

Comment:
I did a bit of digging into why this was needed but couldn't work out why aside from this bug: https://github.com/redux-form/redux-form/issues/686 - which I assume is what you found too.

I found that I also got this problem with the main branch locally as well so I don't think it was something in this code that introduces it.

It'd be good if we put this in a separate commit with some documentation as to why we added it since it's unconventional.

All I can really determine is that when we tell Capybara to fill_in a field it [sends a JS message before it to select() the contents so that the new information replaces the old](https://github.com/teamcapybara/capybara/blob/f7ab0b5cd5da86185816c2d5c30d58145fe654ed/lib/capybara/selenium/node.rb#L281). I'm not sure if we have any events that change focus which perhaps interfere with this somehow and make it a race condition.

Reply:
Good point, have separated this out into a separate commit.